### PR TITLE
add GitHub Pull request template (.md) file

### PR DESCRIPTION
**Pull request by vijaysawant**
### Problem Statement
Most of Pull Requests doesn't have enough information in the description section to go over before starting the review process.
### Solution
Proposed template help streamline the contribution PR generating process by providing guidance and well structured information.

### More information
`.github/new_pull_request.md` template is used to provide guidance and structure to contributors when they create a pull request.

### Related Reference links
[GitHub docs create a PR template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

### Additional Notes
Proposed template has added basic information, request others to modify respective templates as per the requirement.

### CODE REVIEW

File: .github/new_pull_request.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### Problem Statement
+
+
+### Solution
+
+
+### PRT test Cases
+
+
```
Comment (Griffin-Sullivan) [inline .github/new_pull_request.md:9]: I don't think we need this section. If anything we should add a comment only visible when opening the PR that gives an example PRT comment and explains how to use it.

File: .github/new_pull_request.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+### Checklist
+- [ ] I have tested the changes locally.
+- [ ] CherryPick labels added
+
```
Comment (Griffin-Sullivan) [inline .github/new_pull_request.md:13]: I'm not sure we need a check list. The actions will fail if you don't have the cherry-pick label and not all tests need to be ran locally. We mostly care that the test passes in CI.

**Review by Griffin-Sullivan — COMMENTED** [review]
I'm not completely sold on the idea that we need a template. The main benefit would be the `Problem Statement` and `Solution`. Outside of that our main issue is inconsistent contributions which could be solved in refreshing our [contributing documentation](https://github.com/SatelliteQE/robottelo/tree/master/docs).

File: .github/new_pull_request.md (hunk 2 above)
Comment (vijaysawant) [reply .github/new_pull_request.md:13]: Okay, removing this section.

**Review by Griffin-Sullivan — APPROVED** [review]

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**